server/src/test/java/org/elasticsearch/snapshots/MinThreadsSnapshotRestoreIT.java

```diff
@@ -152,58 +152,4 @@ public void testSnapshottingWithInProgressDeletionNotAllowed() throws Exception
         client().admin().cluster().prepareCreateSnapshot(repo, snapshot2).setWaitForCompletion(true).get();
         assertEquals(1, client().admin().cluster().prepareGetSnapshots(repo).setSnapshots("_all").get().getSnapshots(repo).size());
     }
-
-    public void testRestoreWithInProgressDeletionsNotAllowed() throws Exception {
```

Comment:
Removing this one instead of adjusting it, it's totally redundant to the new resiliency test I added, that proves we don't get a dead-lock in this code either and it's much more useful for debugging.